autoload/SpaceVim/layers/lang/c.vim

```diff
@@ -83,6 +83,8 @@
 "
 " }}}
 
+let s:format_on_save = 0
```

Comment:
this line should be moved to the line after `if exists('s:clang_executable') xxx else`